src/components/LandingPage/Carousel/Carousel.jsx

```diff
@@ -0,0 +1,55 @@
+import { Link } from "react-router-dom";
+import "./Carousel.css"
+
+export default function Carousel(){
+   
+    return (
+        <section className="carousel-base grid-carousel carousel-slides">
```

Comment:
Even for carousel, you can have an object and then map over it. Assume I have 10 carousel slides, this will make the code very lengthy.

Reply:
done, thank you!